app/views/entities/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<header class="fixed top-0 w-full">
+  <nav  class="  bg-gradient-to-r from-blue-600 to-blue-900 overflow-hidden flex justify-between items-center w-full p-2 py-4 text-white">
+    <%= link_to image_tag('back.svg', alt: 'Back'), categories_path %>
+    <h1 class="font-light text-xl uppercase">Transactions</h1>
+    <%= button_to destroy_user_session_path, method: :delete,class:"w-6 h-6 m-0 p-0" ,data: { turbo: false } do %>
+      <%= image_tag 'logout.svg', alt: 'Log Out',class:"w-6 h-6 m-0 pt-1"%>
+    <% end %>
+  </nav>
+</header>
+<section class="flex flex-col m-auto p-4 gap-4 bg-[#F0F2F1] pb-32 pt-20 mt-2">
+  <div class="bg-white shadow-sm p-4 pb-0">
+  <article class='flex items-center gap-4'>
+    <% if @category.icon.attached? %>
+          <%= image_tag(@category.icon, class:"w-20 h-20 object-cover")%>
+        <%else%>
+          <%= image_tag "default.jpg", alt:"category icon", class:"w-20 h-20 object-cover"%>
+        <%end%>
+    <h3 class="font-bold text-black text-xl"><%= @category.name %></h3>
+  </article>
+    <div class= "py-4 border-b-2 border-gray-200 ">  
+    <p class='text-sm'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod. Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.</p>
+    </div>
+    <div class='flex justify-between items-cente uppercase py-4  font-bold text-black'>
+    <p>Total payments</p>
+    <p>$ <%=@total_amount%></p>
+  </div>
+
+  <%@category_entities.each do |category_entity|%>
+    <article class='flex flex-col border-t-2 border-gray-200 py-4'>
+      <div class="flex w-full justify-between pb-1 font-bold text-black items-center">
+        <p><%= category_entity.entity.name %></p>
+        <p>$ <%= category_entity.entity.amount %></p>
+      </div>
+      <div class="flex items-center gap-2 text-sm">
+      <%= image_tag 'date.svg', alt: 'Log Out',class:"w-4 h-4 m-0"%>
+      <p><%= category_entity.entity.created_at.strftime('%d %b %Y %H:%M') %></p>
+      </div>
+    </article>
```

Comment:
To make our code base more maintainable, I suggest we make use of partials where necessary as this makes our codebase neater, maintainable, and understandable, and gives it a very professional look👍🏼.

Reply:
@Ibizugbe I understand the point and it is valid if one intended to reuse components with the same style or if I was looking for a scalable application, but since in this case I do not have many pages, and I do not plan to reuse components, since the styles are different, nor scaling I do not see much sense, besides not being a requirement of the project.